### PR TITLE
content iteration

### PR DESCRIPTION
**Pull request by WardenScotland**

### CODE REVIEW

File: _content/support.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
----
 layout: default
+
```
Comment (Acconut) [inline _content/support.md:2]: Can you remove the empty new lines here?

File: _content/implementations.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ These projects are reference implementations maintained by the protocol develope
     </a>
 
     <div class="author">&mdash; by <a href="/support.html">tus</a></div>
-
+
```
Comment (Acconut) [inline _content/implementations.md:37]: In this file, there is a lot of whitespace added. Maybe your editor did that on its own but - if possible - please remove the whitespace again. We have a convention that if a line contains no text, it should also contain no whitespace :)

File: _content/faq.md [hunk 3]
```diff
@@ -18,56 +18,56 @@ Here are some Frequently Asked Questions and their answers. This document is sti
...
 
 ## How does tus work?
 
-This section covers the more low-level details of how tus performs an upload. If you are using one of our open-source [implementations](/implementations.html), you don't have to worry about these details. That being said, they might still come in handy when debugging and troubleshooting.
+This section covers the more low-level details of how tus performs an upload. If you are using one of our open-source [implementations](/implementations.html), you don't have to worry about these details but they may still be helpful to ease debugging and troubleshooting.
 
 A tus upload is broken down into different HTTP requests, where each one has a different purpose:
 
-- At first, the client sends a `POST` request to the server to initiate the upload. This *upload creation request* tells the server basic information about the upload, such as its size or additional metadata. If the server accepts this upload creation request, it will return a successful response with the `Location` header containing the *upload URL*. The upload URL is used to uniquely identify and reference the newly created upload resource.
```
Comment (Acconut) [inline _content/faq.md:55]: In some cases, the backticks (`) were removed. We like to keep them as formatting instruction, so the words in the backticks render as code in the website. Could you undo these deletions? This occurred in multiple files, I think.

**Review by Acconut — CHANGES_REQUESTED** [review]
Wow, thank you very, very much for these impressive changes! It's mostly looking good, expect for some formatting issues

**Comment by WardenScotland** [comment]
Thanks for the notes, Marius. I will address each comment today. I use Typora as my markdown editor and it looks like it messed up some of the file formattings.

**Comment by kvz** [comment]
@WardenScotland can you please revisit this so it's ready for a layer of polish by @AJvanLoon?

**Review by Acconut — APPROVED** [review]

**Comment by Acconut** [comment]
Took care of the changes on my own. Using some RegExp magic, it was quickly done and we can merge this to wipe out my bad spelling from the homepage :D

**Pull request merged**